runtime/commonMain/src/kotlinx/serialization/modules/SerializerAlreadyRegisteredException.kt

```diff
@@ -13,6 +14,7 @@ import kotlin.reflect.KClass
  * Registering in the same module twice is prohibited. In case you are combining modules
  * with [SerialModule.plus], consider using [SerialModule.overwriteWith] if you want overwriting behaviour.
  */
+@InternalSerializationApi // Will be hidden in the next release
```

Comment:
Why so?

Reply:
Because it's one more internal exception without proper use. `IllegalArgumentException` is enough to throw

Reply:
Then you can deprecate it and replace all usages. I doubt anyone actually tried to catch that

Reply:
Let it be for a release, according to the issues, at least one user tried to catch it :)